psi-ms.obo

```diff
@@ -22350,6 +22350,33 @@ name: mzTab-M
 def: "Expanded tabular result format for metabolomics experiments reporting quantitative summary data, MS features and identification evidence." [PMID:30688441, PMID:31525911, https://www.psidev.info/mztab]
 is_a: MS:1000914 ! tab delimited text format
 
+[Term]
+id: MS:1003390
+name: crosslinker cleavage
+def: "Signifies that the crosslinker is cleavable and on cleavage can leave a given stub. The pattern specifies up to three slots <name>:<mass>:<pairs with>." [PSI:PI]
+is_a: MS:1001471 ! peptide modification details
+relationship: has_regexp MS:1003391 ! crosslinker cleavage regular expression
+
+[Term]
+id: MS:1003391
+name: crosslinker cleavage regular expression
+def: "[A-Za-z]:[+-]?[0-9]+(\.[0-9]+)?([eE][+-]?[0-9]+(\.[0-9]+)?)?:[A-Za-z]+" [PSI:PI]
```

Comment:
@colin-combe In your email from 7/12/2023 you stated that you updated the regexp to:
`^(?<NAME>[A-Za-z]):(?<SCORE>[+-]?[0-9]+(\.[0-9]+)?([eE][+-]?[0-9]+(\.[0-9]+)?)?):(?<PAIRS_WITH>[A-Za-z]+)$`
Did this change again or is this outdated here?

Reply:
I guess that should rather be `^(?<NAME>[A-Za-z]):(?<MASS>[+-]?[0-9]+(\.[0-9]+)?([eE][+-]?[0-9]+(\.[0-9]+)?)?):(?<PAIRS_WITH>[A-Za-z]+)$`

Reply:
:+1:  you're right should be `MASS` of course

Reply:
yeah, stupid mistake from me